prime-router/metadata/hl7_mapping/STLTs/CA/ca-specimen.yml

```diff
@@ -18,8 +17,22 @@ elements:
   # Same as above
   - name: ca-specimen-organization-identifier-2
     resource: '%performerOrganization'
-    debug: true
     constants:
       organizationIdFieldPath: '%{hl7SpecimenFieldPath}-2-2'
       identifier: '%diagnostic[%orderIndex].identifier[0].value'
```

Comment:
I don't think I caught this in the code review - I thought we were pulling from the specimen and not the diagnostic report now that SR fixed their unique problem?